plugins/modules/system/parted.py

```diff
@@ -95,6 +95,12 @@
     type: str
     choices: [ absent, present, info ]
     default: info
+  fs_type:
+    description:
+     - If specified, will set filesystem type to given partition
```

Comment:
```suggestion
     - If specified and the partition does not exist, will set filesystem type to given partition.
```
IMO it could be important to know:)

Reply:
Right. But the same note applies to `part_type`, `part_start` and `part_end`.

Reply:
Yep, it can be important for users to know about that as well (you can improve the other options' documentation by a separate PR, not to mix all stuff in this one). Would be good